src/sass/layout/_navigation.scss

```diff
@@ -0,0 +1,177 @@
+        }    
+    }
+}
+
+@media only screen and (max-width: 1024px) {
+    .nav {
+        
+        $hamburger-button-height-width: 2.5rem;
+        $nav-bar-height: 3rem;
+        $hamburger-margin: calc( ($nav-bar-height - $hamburger-button-height-width) / 2);
+        $link-height: 2rem;
+        $link-border-height: 0.063rem;
+        $number-of-menu-links: 5;
+        $menu-display-height: calc($link-height * $number-of-menu-links + $link-border-height);
+        $logo-padding: 0.75rem;
+        $hamburger-bar-height: 0.15rem;
+
+        background-color: var(--navbar-background-color);
+        z-index: 500;
+        padding-top: $nav-bar-height;
+        position: fixed;
+        top: 0;
+    
+        &__logo {
+            position: absolute;
+            display: flex;
+            height: $nav-bar-height;
+            width: fit-content;
+            padding: $logo-padding;
+            top: 0;
+            margin-left: 0;
+            margin-top: 0;
+        }
+    
+        &__hamburger-button {
+            width: $hamburger-button-height-width;
+            height: $hamburger-button-height-width;
+            border-radius: 0.4rem;
+            background-color: var(--hamburger-background-color);
+            display: flex;
+            flex-direction: column;
+            justify-content: space-evenly;
+            align-items: center;
+            position: absolute;
+            top: 0;
+            right: 0;
+            margin-right: $hamburger-margin;
+            margin-top: $hamburger-margin;
+            transition: background 0.15s ease-in-out;
+        }
+    
+        &__hamburger-bar {
+            width: 60%;
+            height: $hamburger-bar-height;
+            border-radius: 1rem;
+            background-color: var(--white-background);
+        }
+    
+        &__menu {
+            display: flex;
+            height: 0;
+            overflow: hidden;
+            margin-top: 0;
+            margin-right: 0;
+            width: 100vw;
+            flex-direction: column;
+            border-top: $link-border-height solid var(--white-border);
+            transition: height 0.15s ease-in-out;
+        } 
+
+        &__list {
+            flex-direction: column;
+            width: 100vw;
+            padding: 0;
+        }
+    
+        &__link {
+            width: 100vw;
+            height: $link-height;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            border-bottom: $link-border-height solid var(--white-border);
+        }
+    
+        &__link:hover {
+            background-color: var(--menu-link-hover-background);
+        }
+
+        &__link:active {
+            background-color: var(--menu-link-hover-background);
+        }
+    
+        &__button {
+            display: none;   
+        }
+
+        //Activated by click with eventlistener in js:
+        //activates the hamburger icon
+        &__menu-display {
```

Comment:
If I understood correctly you can use a modifier like `--visible` on root el here.